pkg/reporter/htmlreporter/package_reports.go

```diff
@@ -36,7 +36,7 @@ type RevisionData struct {
 }
 
 // CreatePackageLevelReports creates the top level report about the package.
-func (r *HTMLReporter) CreatePackageLevelReports(packageNode *service.PackageNode) error {
+func (r *HTMLReporter) CreatePackageLevelReports(packageNode *service.PackageNode, cserv service.ControlServiceClient, rserv service.ReportServiceClient) error {
```

Comment:
We are not using the ControlServiseClient. Do we need it in the signature of the method?